spec/models/spotlight/resources/purl_spec.rb

```diff
@@ -59,7 +62,9 @@
     end
 
     it "should add a document to solr" do
-      expect(subject).to receive(:update_index)
+      solr_data = '[{"spotlight_resource_id_ssim":null,"spotlight_resource_type_ssim":"spotlight/resources/purls","upstream":true}]'
```

Comment:
Instead of building this as a string, what do you think about building a hash and calling `#to_json` on it? To me, it'd be a little easier to read (with syntax highlighting and the like) than as a blob.